src/Raven.Studio/wwwroot/App/views/shell/about.html

```diff
@@ -55,6 +55,22 @@
                 </div>
             </div>
         </div>
+        <div data-bind="visible: passiveNode">
+            <div class="bg-info text-info flex-horizontal padding">
+                <div class="flex-start"><i class="icon-info"></i></div>
+                <div>The running server is in a <strong>Passive State</strong>, it is not part of a cluster yet.<br>
```

Comment:
I'd join last sentence with this one.

Reply:
Refactored text to be:

![image](https://user-images.githubusercontent.com/17376974/99286055-03670300-2841-11eb-9a3b-988deb8d052b.png)